content/rs/clusters/logging/log-security.md

```diff
@@ -16,48 +16,68 @@ aliases: [
 ---
 Redis Enterprise comes with [a set of logs]({{< relref "/rs/clusters/logging/_index.md" >}}) on the server and available through the user interface to assist users in investigating actions taken on the server and to troubleshoot issues.
 
-## Sending logs to a remote logging server
+## Send logs to a remote logging server
 
 Redis Enterprise sends logs to syslog by default. You can send these logs to a remote logging server by configuring syslog.
 
 To do this, modify the syslog or rsyslog configuration on your operating system to send logs in `/var/opt/redislabs/log` to a remote monitoring server of your choice.
 
 ## Log rotation
 
-Redis Enterprise uses the default logrotate daemon to schedule rotation of logs stored on the operating system. The configuration of log rotation may be found at /etc/logrotate.d.
+Redis Enterprise uses the default `logrotate` daemon to schedule rotation of logs stored on the operating system. You can find the log rotation configuration file at `$pkgconfdir/logrotate.conf` as of Redis Enterprise Software version 7.2 (`pkgconfdir` is `/opt/redislabs/config` by default, but can be changed in a custom installation).
```

Comment:
logrotate isn't ran by the default logtorate daemon anymore. In 7.2 out own job scheduler runs it every 5 minutes.

Reply:
Fixed